assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -36,6 +38,14 @@ function HostDetails() {
   const { grafanaPublicUrl } = config;
 
   const [exportersStatus, setExportersStatus] = useState([]);
+  const [cleanUpModalOpen, setCleanUpModalOpen] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const cleanUpHost = () => {
+    setCleanUpModalOpen(false);
+    dispatch(deregisterHost(host));
```

Comment:
We should get the `host` as argument in the function.
If we use the global variable, we have the risk of having strange things.
For example, if the function was used when `host` is not loaded, things wouldn't work.